src/app/containers/CpsAssetPageMain/index.test.jsx

```diff
@@ -66,25 +87,24 @@ describe('CpsAssetPageMain', () => {
       cpsAssetPreprocessorRules,
     );
 
+    await matchSnapshotAsync(createMediaAssetPage({ pageData }));
+  });
+
+  it('should match snapshot for MAP - hidden timestamp', async () => {
```

Comment:
Could you add an expectance test as well as the snapshot

Reply:
Added